endpoints/openrtb2/auction_test.go

```diff
@@ -1089,7 +1161,7 @@ func TestOversizedRequest(t *testing.T) {
 		&mockStoredReqFetcher{},
 		empty_fetcher.EmptyFetcher{},
 		empty_fetcher.EmptyFetcher{},
-		&config.Configuration{MaxRequestSize: int64(len(reqBody) - 1)},
+		&config.Configuration{MaxRequestSize: int64(len(reqBody) - 1), GenerateRequestId: false},
```

Comment:
The nil value of a boolean is `false`, so you shouldn't have to explicitly set these values.